library/std/src/backtrace.rs

```diff
@@ -353,6 +369,22 @@ impl Backtrace {
     }
 }
 
+impl<'a> Backtrace {
+    /// Returns an iterator over the backtrace frames.
+    #[unstable(feature = "backtrace_frames", issue = "79676")]
+    pub fn frames(&self) -> Frames<'a> {
+        if let Inner::Captured(c) = &self.inner {
+            let frames = &c.force().frames;
+            Frames {
+                inner: frames.iter().map(BacktraceFrame::clone).collect(),
```

Comment:
We can actually capture these as a `&'a [BacktraceFrame]` now so don’t need the intermediate collect 🎉